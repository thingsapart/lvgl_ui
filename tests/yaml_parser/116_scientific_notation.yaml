small_number: 1.23e-4
large_number: -5E+10
