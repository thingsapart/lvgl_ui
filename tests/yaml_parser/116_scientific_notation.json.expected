[{
		"small_number":	0.000123,
		"large_number":	-50000000000
	}]
